ninja-core/src/main/java/ninja/standalone/AbstractConsole.java

```diff
@@ -35,13 +38,16 @@
     
     // can all be changed prior to configure()
     protected NinjaMode ninjaMode;
-    protected String externalConfigurationPath;
+    protected Optional<String> externalConfigurationPath = Optional.empty();
     protected String name;
     // internal state
     protected boolean configured;
     protected boolean started;
     protected NinjaPropertiesImpl ninjaProperties; // after configure()
     protected OverlayedNinjaProperties overlayedNinjaProperties; // after configure()
```

Comment:
I have actually not really an idea why we have OverlayedNinjaProperties...

Reply:
I remember implementing it when working on the new standalone class hierarchy.  Your new override stuff may make it obsolete, but we'd have to dive in to the code to be sure.